demo/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>UI Managers</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+<!--    <script src="https://cdnapisec.kaltura.com/p/2215841/embedPlaykitJs/uiconf_id/42237501/"></script>-->
+    <script src="http://localhost:8080/kaltura-ovp-player.js"></script>
+    <script src="./ui-manager.js"></script>
+  </head>
+  <body>
+    <div id="player-placeholder" style="width: 640px; height: 360px"></div>
+    <script type="module">
+      import './some-plugin.js'
```

Comment:
will work w/o webpack?

Reply:
yes it does regardless of Webpack since the script it type module